scripts/globals/mobskills/penta_thrust.lua

```diff
@@ -0,0 +1,31 @@
+---------------------------------------------
+-- Penta Thrust
+-- Delivers a five-hit attack
+-- Type: Physical
+-- Range: Melee
+---------------------------------------------
+require("scripts/globals/mobskills")
+require("scripts/settings/main")
```

Comment:
It doesn't look like any settings are used in this file, so this require isn't needed.